regression/cbmc/havoc_object_slice/declarations.h

```diff
@@ -0,0 +1,26 @@
+#include <stdint.h>
+#include <stdlib.h>
+
+// HAVOC STRUCT MEMBERS
+// TODO take into account plaform/pointer size in the tests
```

Comment:
Create a GitHub issue to track this request and add it to the comment, so it doesn't get lost.

Reply:
If we're not using this file, perhaps add it to the GitHub issue and delete from here instead.